backend/app/model/event.py

```diff
@@ -0,0 +1,29 @@
+import datetime
```

Comment:
Event, Location, and Resource have a lot in common.  I like treating them as different classes but we shouldn't duplicate code.  I recommend we create a parent class for all of them, and it may be the case that event is a special case of location.